upgrading/topics/rhsso/changes-76.adoc~

```diff
@@ -0,0 +1,168 @@
+* After the upgrade, deploy your scripts following the instructions in the https://www.keycloak.org/docs/latest/server_development/#_script_providers[documentation].
+* Update your authentication flows, mappers, and the client authorization settings to use the providers created from the scripts deployed to the server.
+
+=== Account console Patternfly upgrade
+
+The Patternfly (PF) React libraries have been updated updated, `@patternfly/react-core` from v3.153.3 to v4.147.0, `@patternfly/react-icons` from v3.15.16 to v 4.11.8, and `@patternfly/react-styles` from v3.7.14 to v4.11.8. Several minor UI updates were made to bring the account console into alignment with PF design standards.
+
+Custom developed account UIs might not be compatible with these updates due to the breaking changes in PF. Most breaking changes should be resovlable by updating props on PF components.
+
+Resources:
+
+- [Patternfly docs](https://www.patternfly.org)
+
+
+Components known to have breaking changes:
+
+- Alert
+
+  - `action` prop changed to `actionClose`
+
+- Expandable
+
+  - renamed to `ExpandableSection`
+
+- Title
+
+  - size attr now uses `TitleSizes`
+
+- DataListContent
+
+  - `noPadding` changed to `hasNoPadding`
+
+- Grid, Stack, Level, Gallery
+
+  - `gutter` attr changed to `hasGutter`
+
+- Modal
+
+  - sizing control changed from, e.g. `isLarge`, to use `ModalVariant`, e.g. `variant={ModalVariant.large}`
+
+- Select
+
+  - `ariaLabelTypeAhead` to `typeAheadAriaLabel`
+
+  - `isExpanded` to `isOpen`
+
+  - `ariaLabelledBy` to `aria-labelledby`
+
+- DataListContent
+
+  - `noPadding` to `hasNoPadding`
+
+=== Client Policies Migration : client-scopes
+
+If you used a policy including client-scopes condition and edited JSON document directly, you will need to change the "scope" field name in a JSON document to "scopes".
+
+=== Liquibase upgraded to version 4.6.2
+
+Liquibase was updated from version 3.5.5 to 4.6.2, which includes, among other things, several bug fixes, and a new way of registering custom extensions using `ServiceLoader`.
+
+Closely follow the <<_upgrading,Upgrading Guide>>, specifically of *backing up
+existing database before upgrade*. While we did our best to test the consequences of the Liquibase upgrade, some installations could be using specific setup unknown to us.
+
+=== Proxy environment variables
+
+{project_name} now respects the standard `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables for outgoing
+HTTP requests. This change could lead to unexpected use of a proxy server if you have for example the `HTTP_PROXY` variable defined but have no
+explicit proxy mappings specified in your SPI configuration. To prevent {project_name} from using those environment variables,
+you can explicitly create a no proxy route for all requests as `.*;NO_PROXY`.
+
+For more details, see the link:{installguide_link}#_proxy_env_vars[related chapter in the {installguide_name}].
```

Comment:
This was [part of 7.5.1](https://issues.redhat.com/browse/KEYCLOAK-19420) but probably was missing from the  release notes. Not sure if we should mention it for 7.6.0.